src/app/(proper_react)/(onboarding)/user/welcome/View.module.scss

```diff
@@ -51,7 +51,7 @@
     justify-content: center;
 
     @media screen and (min-width: $screen-md) {
-      height: 33%;
+      height: 20%;
```

Comment:
Suggestion: 🪄  sorta Magic Number™! Can you please add a comment for this specific variable?

Reply:
Add a comment in [4d13222](https://github.com/mozilla/blurts-server/pull/3208/commits/4d132222623f3b16a057dc42329be6f2e9b5a620).